Formula/l/lima.rb

```diff
@@ -17,7 +17,7 @@ class Lima < Formula
   end
 
   depends_on "go" => :build
-  depends_on "qemu"
+  depends_on "qemu" if MacOS.version < :ventura
```

Comment:
vz is still experimental and using qemu is recommended

Reply:
On M1 and M2, qemu seems actually more stable
- https://github.com/lima-vm/lima/issues/1957#issuecomment-1812631707